LedRGB.cpp

```diff
@@ -0,0 +1,35 @@
+#include "LedRGB.h"
+#include "Led.h"
+
+LedRGB::LedRGB() : RedLed(0), GreenLed(0), BlueLed(0) {
+}
+
+LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
+    : RedLed(redPin), GreenLed(greenPin), BlueLed(bluePin) {
+    //	pinMode(redPin, OUTPUT);
+    //	pinMode(greenPin, OUTPUT);
+    //	pinMode(bluPin, OUTPUT);
+}
+
+void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
+    //	analogWrite(redPin, r);
+    //	analogWrite(greenPin, g);
+    //	analogWrite(bluePin, b);
```

Comment:
Now that we have Led objects we want to use them instead of directly setting the voltages. Same goes for every function in the class. No arduino functions should be necessary